packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/BotMessage.tsx

```diff
@@ -2,9 +2,39 @@ import React from 'react';
 import Message from '@patternfly/chatbot/dist/dynamic/Message';
 import patternflyAvatar from './patternfly_avatar.jpg';
 import squareImg from './PF-social-color-square.svg';
+import { FormGroup, Radio } from '@patternfly/react-core';
```

Comment:
```suggestion
import { Form, FormGroup, Radio } from '@patternfly/react-core';
```

can we wrap the radio buttons/form groups in a form component? I _believe_ that should add a little more padding/spacing between each!

Reply:
Seems like it works a lot better 👍